Scripts/compile_demo_apps.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# source: https://stackoverflow.com/a/3352015/4080860
+trim() {
+    local var="$*"
+    # remove leading whitespace characters
+    var="${var#"${var%%[![:space:]]*}"}"
+    # remove trailing whitespace characters
+    var="${var%"${var##*[![:space:]]}"}"   
+    echo -n "$var"
+}
+
+build() {
+    xcodebuild build                       \
+        -workspace Demos/Demos.xcworkspace \
```

Comment:
Use one style. I suggest to wrap every path in "" since path can contain whitespaces. You do so in Tests script.